packages/actions/score/src/components/Scoreboard/Scoreboard.spec.js

```diff
@@ -0,0 +1,99 @@
+import {render, screen} from '@testing-library/react'
+
+import Scoreboard from './Scoreboard'
+
+describe('actions', () => {
+  describe('score', () => {
```

Comment:
suggestion(non-blocking): usually we try to follow the folder path here

In this case it would be:
```
describe('score'), () => {
    describe('components'), () => {
        describe('Scoreboard'), () => {

        })
    })
})
```
It's a bit strange because we omit the `actions`|`core`|`apps`|`widgets` folder because this was added after we introduced the tests and we did not want to re-write all the tests.
Also we omit the `src` folder because it does not give us any further information.